legend-depot-core-tracing/src/main/java/org/finos/legend/depot/tracing/resources/ResourceLoggingAndTracing.java

```diff
@@ -47,6 +47,7 @@
     public static final String UPDATE_VERSION = "refresh version";
     public static final String UPDATE_ALL_PROJECT_VERSIONS = "refresh all project versions";
     public static final String EVICT_OLD_VERSIONS = "evict old versions";
+    public static final String DELETE_VERSION = "permanently delete version for ga";
```

Comment:
can we make this shorter as its used in prom metrics.. just call it delete version

Reply:
done